datalad/cmd.py

```diff
@@ -262,20 +273,36 @@ def call(self, f, *args, **kwargs):
         *args, **kwargs:
           Callable arguments
         """
-        if self.dry:
-            self.commands.append("%s args=%s kwargs=%s" % (f, args, kwargs))
+        if self.protocol.do_execute_callables:
+            if self.protocol.records_callables:
+                prot_exc = None
+                prot_id = self.protocol.start_section([str(f),
+                                                 "args=%s" % args.__str__(),
+                                                 "kwargs=%s" % kwargs.__str__()])
```

Comment:
if you are to convert explicitly, don't use `__str__` -- call  `str(x)`.  But theoretically not needed since you use %s, so I would have done  ```% (args,)``` -- save characters ;)